project/PATCH-RELEASES.md

```diff
@@ -0,0 +1,53 @@
+# Docker patch (bugfix) release process
+
+Patch releases (the 'Z' in vX.Y.Z) are intended to fix major issues in a
+release. Docker open source projects follow these procedures when creating a
+patch release;
+
+After each release (both "major" (vX.Y.0) and "patch" releases (vX.Y.Z)), a
+patch-release milestone (vX.Y.Z + 1) is created.
```

Comment:
@thaJeztah what we could do, is be more explicit in the description of the milestones, just like Go does: https://github.com/golang/go/milestones. There's a whole paragraph explaining that Go1.4.4 will likely never happen, but it's still a milestone. Same for Go 1.5.3: `Go 1.5.3, if that happens.`

So maybe it's fine to keep the document the way you have it now.